docs/make.jl

```diff
@@ -35,16 +35,22 @@ import Pkg
 Pkg.status()
 """
 
-example_pages = [
-    "Stratified dry thermal bubble" => "literated/dry_thermal_bubble.md",
-    "Cloudy thermal bubble" => "literated/cloudy_thermal_bubble.md",
-    "Cloudy Kelvin-Helmholtz instability" => "literated/cloudy_kelvin_helmholtz.md",
-    "Shallow cumulus convection (BOMEX)" => "literated/bomex.md",
-    "Precipitating shallow cumulus (RICO)" => "literated/rico.md",
-    "Convection over prescribed sea surface temperature (SST)" => "literated/prescribed_sea_surface_temperature.md",
-    "Inertia gravity wave" => "literated/inertia_gravity_wave.md",
-    "Single column gray radiation" => "literated/single_column_radiation.md",
-    "Stationary parcel model" => "literated/stationary_parcel_model.md",
+struct Example
+    title::String
+    filename::String
```

Comment:
if we define a struct, and if this is "filename" then it makes sense to expect the `example.jl`, not the `.md`?

Reply:
true